egs2/owsm_ctc_v3.1/s2t1/conf/train_s2t_multitask-ctc_ebf27_conv2d8_size1024.yaml

```diff
@@ -0,0 +1,109 @@
+preprocessor: s2t_ctc
```

Comment:
Can you add the number of GPUs and GPU types and how long training took?

Reply:
Sure. Added